include/cpuinfo_aarch64.h

```diff
@@ -62,4 +62,8 @@ const char* GetAarch64FeaturesEnumName(Aarch64FeaturesEnum);
 
 CPU_FEATURES_END_CPP_NAMESPACE
 
+#if !defined(CPU_FEATURES_TEST) && !defined(CPU_FEATURES_ARCH_AARCH64)
+#error "Including cpuinfo_aarch64.h from a non-aarch64 architecture."
```

Comment:
s/from a non-aarch64/for a on-aarch64 targeted/
I mean you can compile *from a* x86_64 host *for a* aarch64 target